rectools/models/sasrec.py

```diff
@@ -592,26 +597,27 @@ class SasRecModel(ModelBase):
 
     def __init__(  # pylint: disable=too-many-arguments
         self,
-        session_maxlen: int,
-        lr: float,
-        batch_size: int,
-        epochs: int,
-        device: str,
-        n_blocks: int,
-        n_factors: int,
-        n_heads: int,
-        dropout_rate: float,
-        use_pos_emb: bool = True,
+        n_blocks: int = 1,
+        n_heads: int = 1,
+        n_factors: int = 128,
+        use_pos_emb: bool = False,
```

Comment:
Default should be True because it is a classic part of SASRec

Reply:
fixed